### ISSUE TITLE
[UI-Update] Settings

### ISSUE THREAD

**Issue by Bastien-Satochip** [Toporin/Seedkeeper-iOS#81]
Could you update the Settings page: 

1) Regarding the reset, we have spoken about how to make it clear on a user point of view. 
The current proposal is not straight forward. The user do not know that the red square is a button. 

Could you try to adapt the UI from this: 
![image](https://github.com/user-attachments/assets/8d6bd19c-7484-4cfe-9da6-49f58063c965)

To this: 
![image](https://github.com/user-attachments/assets/181662ba-4aaa-4896-b8e0-9acc0bee1708)

In other words: 

- The title is: 
-- FR: Réinitialisation d'usine
-- EN: Factory reset

- The sub is:
-- FR: Réinitialiser votre carte dans son état d'usine.
-- EN: Reinitialize your card to its factory state.

- Red warning message:
-- FR: Attention : cette opération supprimera tous vos secrets !
-- EN: Warning: this will erase all your secrets!

And the button: 
-- FR: Réinitialiser ma carte
--EN: Factory reset my card
